### DISCUSSION TITLE
https://imagekit.io/docs/api-reference/digital-asset-management-dam/list-and-search-assets

### DISCUSSION THREAD

**Discussion by giscus** [imagekitio/discussions#55] [Docs]
# https://imagekit.io/docs/api-reference/digital-asset-management-dam/list-and-search-assets

This API can list all the uploaded files and folders in your ImageKit.io media library. In addition, you can fine-tune your query by specifying various filters by generating a query string in a Lucene-like syntax and provide this generated string as the value of the `searchQuery`.

https://imagekit.io/docs/api-reference/digital-asset-management-dam/list-and-search-assets

**Comment by iam11thsiddharth** [comment]
(:) is to be added at last in your existing private Api key which is required in Authorization header. Small addition but higher chances of overlooking, please highlight.
